### ISSUE TITLE
Videos throw adsloader error and don't autoplay

### ISSUE THREAD

**Issue by gtrufitt** [guardian/frontend#10672]
The videos linked from http://www.theguardian.com/lexus-create-amazing are set-up to auto-play in the mark-up, as per other non-commercial video content. When these pages load, however, they log an error related to the video player expecting an ad playlist which seems to prevent the auto-playing of the video.

The error is: AdsLoader error: AdError 1010: The ad playlist response was malformed or empty. which refers to [google.ima.AdError](https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdError.Type)

Testing locally, I can add the following in the [onAdsLoaderError_ listener](https://github.com/guardian/frontend/blob/157d7230cd1dbc1837ddb8d3c2b55de50862234d/static/src/javascripts/components/videojs-ima/videojs.ima.js#L219)

```
adContainerDiv.style.display = 'none';
      player.play();
```

as per [onAddError_ listener](https://github.com/guardian/frontend/blob/157d7230cd1dbc1837ddb8d3c2b55de50862234d/static/src/javascripts/components/videojs-ima/videojs.ima.js#L231) which seems to be similar to what [newer version of the script from Google does](https://github.com/googleads/videojs-ima/blob/master/src/videojs.ima.js#L258).

This masks the error, but obviously doesn't fix it - @sndrs @desbo @OliverJAsh - What are your thoughts on adding this in so that the autoplaying of ads throwing the AdsLoader still load as expected, while I investigate the root issue? Do you have any ideas on the root issue..?

This can also be seen on [this video](http://www.theguardian.com/science/video/2015/sep/28/super-blood-moon-red-lunar-eclipse-around-world-video)

**Comment by Calanthe** [comment]
I think that it is a known issue https://github.com/googleads/videojs-ima/issues/24

**Comment by gtrufitt** [comment]
Thanks @Calanthe - That looks like a better option using the method described there:

```
vjsplayer.on("adtimeout", function (e) {
  this.ready(function () {
    this.play();
  });
});
```

rather than modifying the actual script (looks like they acknowledged the issue and updated the script later on anyway).

What's still unclear to me is if we should ever be getting that adError or whether that is expected in our case.

**Comment by Calanthe** [comment]
That adError is (afaik) fine. It is a way to inform us that there is no video pre-roll servered by DFP.

**Comment by gtrufitt** [comment]
Ok, cheers. I'll test the ``adtimeout`` event.

**Comment by gtrufitt** [comment]
Updating to latest version of video.ima.js fixes this bug, we need to test whether it fixes #10034, however, there have been some modifications to our version of the video.ima.js file, so would need a full QA on all versions of video player if replacing directly, otherwise my initial comment is the basic fix for this bug.
